apps/communications/contacts/js/match_service.js

```diff
@@ -9,6 +9,13 @@
   var extensionFrame = document.createElement('iframe');
   extensionFrame.id = 'match-extension';
   document.body.appendChild(extensionFrame);
+
+  if (document.querySelector('#fb-curtain') == null) {
```

Comment:
Create a bug in order to remove all 'fb-'  prefix and add a TODO with bug number, just to keep tracking of all pending tasks to do.